numpy/core/tests/test_umath_accuracy.py

```diff
@@ -5,13 +5,15 @@
 import pytest
 from ctypes import c_float, c_int, cast, pointer, POINTER
 from numpy.testing import assert_array_max_ulp
+from numpy.core._multiarray_umath import __cpu_features__
 
-runtest = sys.platform.startswith('linux') and (platform.machine() == 'x86_64')
+IS_AVX = __cpu_features__.get('AVX512F', False) or \
+        (__cpu_features__.get('FMA3', False) and __cpu_features__.get('AVX2', False))
+runtest = sys.platform.startswith('linux') and IS_AVX
 platform_skip = pytest.mark.skipif(not runtest,
                                    reason="""
-                                   stick to x86_64 and linux platforms.
-                                   test seems to fail on some of ARM and power
-                                   architectures.
+                                   avoid testing inconsistent platform
+                                   library implementations
```

Comment:
would probably not indent `reason` the full way, as to have a lot of space for the message. Would also be better with string continuation:
```
"asdf string "
"continued on next line"
```
this way, the indentation and newline is actually printed.